vuepress/docs/next/tutorials/ecr/deploy-components-without-bundle.md

```diff
@@ -1,46 +1,54 @@
 ---
 redirectFrom: /next/tutorials/ecr/tutorials/ecr-deploy-use-plugin-and-mfe-without-bundle.html
 ---
-# Use Blueprint Generated Plugin and Micro Frontends Without a Bundle
+# Apply Entando Blueprint Without a Bundle
 
 ## Overview
 
-In this tutorial you will learn how to use the plugin and microfrontend generated using the Entando Blueprint in a running cluster without the need to include the components in a bundle.
-
-**If you haven't already generated a plugin and micro frontends with the Component Generator go here first [Generate a Plugin](../backend-developers/generate-microservices-and-micro-frontends.md)**
+In this tutorial you will learn to apply an Entando Blueprint to a plugin and Micro Frontend generated with the Entando Component Generator in a running cluster, without the need for a bundle.
 
 ## Prerequisites
 
--   An Entando Plugin built with the Entando Component Generator and populated with micro frontends
+-   An Entando Plugin built with the Entando Component Generator and populated with Micro 
```

Comment:
ack!! plugin or Plugin??

Reply:
I think plugin since it's not a feature per se. Seems like this page is capitals-happy even more than others...

Reply:
plugin, I think.

Reply:
I think there is one that is built with the Entando component Generator that is specific to Entando so capped. In most other cases, lowercase is okay I think but will have to think on this a bit